src/shared/components/food-truck-card/components/ReservationInfos.tsx

```
@@ -0,0 +1,33 @@
+        <span className='body-s-11 text-grayscale-700'>장소</span>
+        <span className='caption-m-12 text-grayscale-700'>{address}</span>
+      </div>
+
+      <div className='flex gap-[0.8rem]'>
+        <span className='body-s-11 text-grayscale-700'>일시</span>
+        <div className='flex flex-col'>
+          {dateTimeInfos.map((dateTimeInfo) => {
+            const [startDateShort, endDateShort, startTime, endTime] = formatDateTimeInfos(dateTimeInfo);
+            return (
+              <div className='flex items-center justify-start gap-[0.2rem]'>
+                <span className='caption-m-11 text-grayscale-700'>{startDateShort} - {endDateShort}</span>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)